nautobot/ipam/api/serializers.py

```diff
@@ -160,11 +161,37 @@ class Meta:
             "description",
         ]
         extra_kwargs = {
-            "ip_version": {"read_only": True},
+            "ip_version": {"read_only": True, "label": "IP Version"},
```

Comment:
Did you try setting on the model field instead of the serializer? I think that's how `vlan` and `rir` are currently set up on Prefix and it looks like those are working in your screenshot without needing this here.

Reply:
I see yeah setting `verbose_name` worked!